modmesh/app/euler1d.py

```diff
@@ -538,33 +693,13 @@ def single_layout(self):
 
     def grid_layout(self):
         """
-        Toolbar action callback that switch plot holder to grid plot layout.
+        Button action callback that switch plot holder to grid plot layout.
 
         :return: None
         """
         self.use_grid_layout = True
         self.plot_holder.plot = self.build_grid_figure()
 
-    def save_file(self):
-        """
-        Toolbar action callback, that use pixmap to store plotting area
-        and open a dialog that allows user to decide where to store the
-        figure file.
-
-        :return: None
-        """
-        fig = QPixmap(self.plot_holder.plot.size())
-        self.plot_holder.plot.render(fig)
-
-        options = QFileDialog.Options()
-        options |= QFileDialog.DontUseNativeDialog
-        fileName, _ = QFileDialog.getSaveFileName(None, "Save file", "",
-                                                  "All Files (*)",
-                                                  options=options)
-
-        if fileName != "":
-            fig.save(fileName, "JPG", 100)
-
```

Comment:
Using matplotlib built-in toolbar, it also contain save file button
so remove self-development save file callback.